q2_feature_table/_summarize.py

```diff
@@ -14,6 +14,54 @@
 import scipy.optimize
 import seaborn as sns
 import matplotlib.pyplot as plt
+from q2_types.feature_data import DNAIterator
+
+_blast_url_template = ("http://www.ncbi.nlm.nih.gov/BLAST/Blast.cgi?"
+                       "ALIGNMENT_VIEW=Pairwise&PROGRAM=blastn&DATABASE"
+                       "=nt&CMD=Put&QUERY=%s")
+
+
+def view_seq_data(output_dir: str, data: DNAIterator) -> None:
+    with open(os.path.join(output_dir, 'index.html'), 'w') as fh:
+        fh.write('<html><body>\n')
+        # style adapted from http://stackoverflow.com/a/6667942/3424666
+        fh.write(' <style>\n'
+                 '  table {table-layout:fixed; width:1250px;}\n'
+                 '  table td {word-wrap:break-word;}\n'
+                 ' </style>\n')
+        fh.write('To BLAST a sequence against the NCBI nt database, click the '
+                 'BLAST link next to that sequence and then click the '
+                 '<i>View report</i> button on the resulting page.<p>\n')
+        fh.write(' <table border=1>\n')
+        fh.write('  <tr><td width=300px>Feature ID</td>'
+                 '<td width=100px>BLAST link</td>'
```

Comment:
These `td` are called `th` (for table-header) nowadays, but it doesn't really matter. There would also be a `tbody` and `thead` section, but browsers can figure it out.